tests/parity/parser/vllm.py

```
@@ -251,21 +255,76 @@ def parse(
     try:
         parser_cls = ToolParserManager.get_tool_parser(key)
         # vLLM's ToolParser constructor checks `if not self.model_tokenizer:` and raises
         # if falsy. None of the parsers we test actually call tokenizer methods inside
         # extract_tool_calls(), so a truthy stub satisfies the check.
-        parser: ToolParser = parser_cls(tokenizer=_StubTokenizer(), tools=wrapped_tools)
+        parser: ToolParser = parser_cls(
+            tokenizer=cast(TokenizerLike, _StubTokenizer()),
+            tools=wrapped_tools,
+        )
+        return parser, wrapped_tools, None
+    except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)